src/twoplayermode/TwoPlayerMode.java

```diff
@@ -26,7 +27,7 @@
 public class TwoPlayerMode extends GameScreen {
 
 
-    public int livestwo = 3;
+    public static int livestwo = 3;
```

Comment:
I noticed that getter and setter methods have been added, but I’m curious why the livestwo field is set to public and static.

Reply:
> I noticed that getter and setter methods have been added, but I’m curious why the livestwo field is set to public and static.

Modifications were made to synchronize the liveestwo values between the GameScreen and TwoPlayerMode classes and to manage them identically.

Reply:
> I noticed that getter and setter methods have been added, but I’m curious why the livestwo field is set to public and static.

I wanted to do it using the instance field without using static, but I did it using static because I thought I would have to modify the code of another team to do so.
Should I modify the code to use the instance field?

Reply:
> > I noticed that getter and setter methods have been added, but I’m curious why the livestwo field is set to public and static.
> 
> I wanted to do it using the instance field without using static, but I did it using static because I thought I would have to modify the code of another team to do so.

Reply:
It seems fine for now since there are no issues

Reply:
> It seems fine for now since there are no issues

Thank you.